src/sortingtraining/SortTester.java

```diff
@@ -59,21 +47,37 @@ private void mainMenu(Scanner in, int[] numbers) {
 		}
 	}
 
+	private Strategy getStrategy(Scanner in) {
+
+		Strategy strategy;
+		while (true) {
+
+			System.out.println(
+					"Type a sorting strategy(" + Strategy.list() + "):\n");
+			try {
+				strategy = Strategy.valueOf(in.nextLine().toUpperCase());
```

Comment:
you can safely change this line to: 
`return Strategy.valueOf(in.nextLine().toUpperCase());`

Reply:
Done, actually implemented also with method getInt().